pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -92,9 +93,8 @@ func (i *DBIngestor) parseData(tts []prompb.TimeSeries, req *prompb.WriteRequest
 			return nil, rows, errors.ErrNoMetricName
 		}
 		sample := model.SamplesInfo{
-			Labels:   seriesLabels,
-			SeriesID: -1, // sentinel marking the seriesId as unset
-			Samples:  t.Samples,
+			Labels:  seriesLabels,
+			Samples: t.Samples,
```

Comment:
(not in this PR) Instead of making a new local allocation, should SamplesInfo struct just extend the `promb.Timeseries` with just our `Labels`? This will save from local allocation, especially in high ingestion with more samples count.

Reply:
I think extensions are just syntactic sugar for writing 
```
type myType{
  typeImExtending
  somethingElse OfType
}
```
The alloc will happen

Reply:
I mean, since the incoming ts are already allocated, so something like

```
type PSSeries struct {
  *promb.Samples
  labels *Labels
}
```

then using it as

```
&PSSeries{&t.Samples, seriesLabels}
```

will this be any good?

Reply:
I wrote this assuming it was all pointers coming, but now I see them as not. So, just in case.

Reply:
Also t.Samples is a slice so what we are doing is just copyng a pointer